libs/tokens/src/hooks/tokens/useAreThereTokensWithSameSymbol.ts

```diff
@@ -2,17 +2,20 @@ import { useAtomValue } from 'jotai'
 import { useCallback } from 'react'
 
 import { isAddress } from '@cowprotocol/common-utils'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
 
 import { tokensBySymbolAtom } from '../../state/tokens/allTokensAtom'
 
-export function useAreThereTokensWithSameSymbol(): (tokenAddressOrSymbol: string | null | undefined) => boolean {
+export function useAreThereTokensWithSameSymbol() {
   const tokensBySymbol = useAtomValue(tokensBySymbolAtom)
 
   return useCallback(
-    (tokenAddressOrSymbol: string | null | undefined) => {
+    (tokenAddressOrSymbol: string | null | undefined, chainId: SupportedChainId) => {
       if (!tokenAddressOrSymbol || isAddress(tokenAddressOrSymbol)) return false
 
-      const tokens = tokensBySymbol[tokenAddressOrSymbol.toLowerCase()]
+      if (tokensBySymbol.chainId !== chainId) return false
```

Comment:
Got rid of false poisitive "There is more than one token in the list" warning displaying